query/expr/ast.go

```diff
@@ -53,6 +53,26 @@ var typeNames = map[Type]string{
 	GeoShape:    "GeoShape",
 }
 
+// constants for call names.
+const (
+	ConvertTzCallName           = "convert_tz"
+	CountCallName               = "count"
+	DayOfWeekCallName           = "dayofweek"
+	FromUnixTimeCallName        = "from_unixtime"
+	GeographyIntersectsCallName = "geography_intersects"
+	HexCallName                 = "hex"
+	// hll aggregation function applies to hll columns
+	HllCallName = "hll"
+	// countdistincthll aggregation function applies to all columns, hll value is computed on the fly
+	CountDistinctHllCallName = "countdistincthll"
+	HourCallName             = "hour"
+	ListCallName             = ""
```

Comment:
empty?

Reply:
yeah that's when we get `("foo", "bar")` in the query